turbinia/turbiniactl.py

```diff
@@ -223,6 +223,9 @@ def main():
   # Server
   subparsers.add_parser('server', help='Run Turbinia Server')
 
+  #print version
+  log.info('version: {0:s}'.format(__version__))
```

Comment:
Isn't this the same as the previous PR?  Also, just FYI, you shouldn't have to delete/recreate the Pull Requests, you should be able to push a commit to the same branch, and it will be reflected in the PR, :).

Reply:
does it? oops